stlab/future.hpp

```diff
@@ -1042,24 +1047,33 @@ namespace detail
             , _holds(_remaining)
         {}
 
-        std::atomic_size_t                    _remaining{0};
+        std::atomic_size_t                    _remaining;
```

Comment:
why remove the initializer? I believe this is no unitialized (the default ctor on atomics are like the default ctors on an int - they don't do anything).